spec/liquid_tags/unified_embed/registry_spec.rb

```diff
@@ -161,6 +161,20 @@
       end
     end
 
+    it "returns OpenGraphTag for pathless or invalid Forem url", :aggregate_failures do
+      # for localhost, the first 2 cases are identical, due to no dots in URL.url
+      invalid_forem_links = [
+        URL.url, # pathless domain
+        URL.url.tr(".", "n"),
```

Comment:
nitpick: one situation not tested here but expected to result in the OG tag is `"#{URL.url}/"` (called out in the QA notes, not part of the tested cases).